Fix/asyncrender closure (#540)
V4Fire/Client: CHANGELOG.md

_Note: Gaps between patch versions are faulty, broken or test releases._

## v3.0.0-rc.194 (2021-05-28)

#### :bug: Bug Fix

> * Fixed bug with referencing value from `closure` in `AsyncRender.iterate`

Reviewer: ```suggestion
* Fixed a bug with referencing a closure' value `AsyncRender.iterate`
```
